tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/mapper/TaskMapper.java

```diff
@@ -43,7 +43,11 @@ public TaskSearchResponse toTaskSearchResponse(final TaskDTO taskDTO) {
             .setTaskState(taskDTO.getTaskState())
             .setSortValues(taskDTO.getSortValues())
             .setIsFirst(taskDTO.getIsFirst())
-            .setFormKey(taskDTO.getFormKey())
+            .setFormKey(
```

Comment:
the same needs to be done in `TaskResponse toTaskResponse(TaskDTO)` below

Reply:
On V2 yes, on V1 I am reproducing the same behaviour as Job Workers (where set the external form on formKey), where if there is external form, this is set as the form key. Otherwise the developer would need create a condition on their side to check it, considering in 8.6 he has support still for both Job Workers, and Zeebe User Tasks.